common4j/src/main/com/microsoft/identity/common/java/eststelemetry/LastRequestTelemetry.java

```diff
@@ -42,6 +43,8 @@
     @SerializedName("failed_requests")
     private List<FailedRequest> failedRequests;
 
+    @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE",
+            justification = "We get this warning because super() also have a @NonNull check inserted by lombok.")
```

Comment:
Is the check injected into the constructor by lombok? Should we just remove the annotation here?

Reply:
We want NonNull to be in effect when both constructors are called.

Reply:
(I'm more inclined to suppress this warning for the whole project instead of removing Nonnull annotation, if we were to do it)

Reply:
yes, if there was an easy way to just do it once for whole project then let's do that

Reply:
@iamgusain  do you know if there's a way to suppress a certain warning?

Reply:
https://github.com/AzureAD/android-complete/pull/92